src/Tests/Microsoft.NET.Sdk.WorkloadManifestReader.Tests/SdkDirectoryWorkloadManifestProviderTests.cs

```diff
@@ -293,6 +279,82 @@ var sdkDirectoryWorkloadManifestProvider
                 .BeEquivalentTo("6.0.100", "5.0.100");
         }
 
+        [Fact]
+        public void ItShouldFallbackWhenPreviewFeatureBandHasNoManifests()
```

Comment:
I'd suggest also having a test for rolling back between preview bands, IE where the SDK is `6.0.100-preview.7` but it can pick up a workload manifest from the `6.0.100-preview.6` band.

Reply:
@joeloff I'm noticing that ReleaseVersions treat different previews as equal (for example, 6.0.100-preview.4 == 6.0.100-preview.5). Is that expected? Do we have any libraries that know how to determine the precedence of all prereleases (ie rc > preview > alpha) or is that something I'll need to implement by hand here?

Reply:
`NuGetVersion` does a full semantic version comparison, I believe.

Reply:
The ReleaseVersion should comply with SemVer 2.0.

Reply:
Looks like there's a bug in ReleaseVersion, @joeloff says he has a PR out to fix but for now I'm using NugetVersions for cross preview comparisons.

Reply:
Can one of you link the ReleaseVersion fix PR?

Reply:
Sorry meant to say he's putting a PR out, there's not a link yet.

Reply:
https://github.com/dotnet/deployment-tools/pull/158